pkg/reconciler/nscert/nscert.go

```diff
@@ -198,3 +202,8 @@ func findNamespaceCert(ns *corev1.Namespace, certs []*v1alpha1.Certificate) (*v1
 	}
 	return nil, apierrs.NewNotFound(v1alpha1.Resource("certificate"), ns.Name)
 }
+
+func certDeepEqual(existingCert, desiredCert *v1alpha1.Certificate) bool {
```

Comment:
Please add comments to `certDeepEqual` to make it clear why comparing the labels is important here.

Reply:
After thinking about this more I don't think we actually need this check at all. Because of the way we're checking for existing certificates (https://github.com/knative/serving/blob/master/pkg/reconciler/nscert/nscert.go#L115) a change in only the label won't cause the certificate to be re-reconciled.